Zend/tests/debug_info/debug_info-error-0.0.phpt

```diff
@@ -17,4 +17,4 @@ $c = new C(0.0);
 var_dump($c);
 ?>
 --EXPECTF--
-Fatal error: __debuginfo() must return an array in %s%eZend%etests%edebug_info-error-0.0.php on line %d
+Fatal error: __debuginfo() must return an array in %s%eZend%etests%edebug_info%edebug_info-error-0.0.php on line %d
```

Comment:
Normally we don't check file names except for various particular cases, so I would just make the whole filename be a `%s`

Reply:
Good to know, will remember that when I write tests in the future. Removing